### DISCUSSION TITLE
Neuen zusätzlichen HMS800 erhalten SN nicht einrichtbar...

### DISCUSSION THREAD

**Discussion by do9da** [tbnobody/OpenDTU#2179] [Q&A]
moin zusammen, habe heute einen zus. HMS800 mit einer SN bestehend aus Zahlen und Buchstaben bekommen. Leider sind beim Einrichten keine Buchstaben in der ODTU eingebbar! Ein HMS 1800 läuft allerdings (SN ohne Buchstaben) sowie ein HM1500 und ein HM400! Ich bekomme eine Meldung der DTU daß es ein Update gibt, finde allerdings keinen Syntax darüber. Hier die vorh. Daten:
Vorab schon mal Danke!
Hostname | OpenDTU-2EDAC0
-- | --
v4.4.4
0.1.25
v23.12.19
opendtufusionv2

OpenDTU-2EDAC0
SDK-Version	v4.4.4
Konfigurationsversion	0.1.25
Firmwareversion / git Hash	[v23.12.19](https://github.com/tbnobody/OpenDTU/releases/tag/v23.12.19)
PIO Umgebung	opendtufusionv2

**Comment by stefan123t** [comment]
Aktuell noch nicht im master branch. Eventuell workaround über Export / Import der config.json

**Comment by tbnobody** [comment]
> Aktuell noch nicht im master branch. Eventuell workaround über Export / Import der config.json

Das ist so nicht richtig. Die funktion für Buchstaben in der Seriennummer ist seit ca. 5 Monaten enthalten

**Reply by do9da** [reply]
exakt! mein HMS1800 ist 10 Mon alt, ohne Buchstaben. Die Buchstaben werden beim Versuch einzutragen einfach ignoriert, Zahlen hingegen ok!

**Reply by tbnobody** [reply]
Was heißt exakt? Ich meinte, dass das feature zum Eingeben der Buchstaben seit ca. 5 Monaten in OpenDTU enthalten ist.

**Reply by do9da** [reply]
dann hab ich das falsch verstanden... scheinbar ist meine DTU dann mit einer zu alten FW versehen...

**Reply by tbnobody** [reply]
ja. deine firmware ist vom 19. dezember 2023
